src/clients/UBAClient/UBAClientWithRefresh.ts

```diff
@@ -25,38 +25,26 @@ export class UBAClientWithRefresh extends BaseUBAClient {
    * Updates the clients and UBAFeeCalculators.
    * @param forceRefresh An optional boolean to force a refresh of the clients.
    */
-  public async update(state: { [chainId: number]: UBAChainState }, forceClientRefresh?: boolean): Promise<void> {
-    if (state) {
-      await super.update(state);
-      return;
-    }
-    // Update the clients if the necessary clients have not been updated at least once.
-    // Also update if forceClientRefresh is true.
-    if (forceClientRefresh || !this.isUpdated) {
-      // Update the Across config store
-      await this.hubPoolClient.configStoreClient.update();
-      // Update the HubPool
-      await this.hubPoolClient.update();
-      // Update the SpokePools
-      await Promise.all(Object.values(this.spokePoolClients).map(async (spokePoolClient) => spokePoolClient.update()));
-    }
-    return this.update(
-      await updateUBAClient(
-        this.hubPoolClient,
-        this.spokePoolClients,
-        this.chainIdIndices,
-        this.tokens,
-        forceClientRefresh,
-        this.maxBundleStates
-      )
-    );
+  public async update(state: UBAClientState, forceClientRefresh?: boolean): Promise<void> {
```

Comment:
It's confusing to only use `forceClientRefresh` if `state` is undefined.

Reply:
Followed up offline

Reply:
I'm skeptical about the UX of this, but you're right that it was pre-existing, so fair enough :+1: